ee/packages/git-sync-manager/src/models.ts

```diff
@@ -836,7 +836,10 @@ export type EnumResourceTypeFilter = {
 };
 
 export enum EnumSchemaNames {
```

Comment:
This comment is optional for your consideration. I think that EnumSchemaNames is too generic. I would rename it to EnumPreviewUserSchemaNames or any other naming idea that explains what are those schema names.